specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,3029 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Cognitive Language Service",
+    "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>.0",
+    "version": "2022-02-01-preview"
+  },
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "description": "A subscription key for a Language service resource.",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/language",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "paths": {
+    "/:analyze-text": {
+      "post": {
+        "summary": "Request text analysis over a collection of documents.",
+        "description": "Submit a collection of text documents for analysis.  Specify a single unique task to be executed immediately.",
+        "operationId": "AnalyzeText",
+        "consumes": [
+          "application/json",
+          "text/json"
+        ],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ShowStats"
+          },
+          {
+            "description": "Collection of documents to analyze and a single task to execute.",
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeTextTask"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeTextTaskResult"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```

Comment:
Could we pull forward and use the common.json we created for the Language pillar? This has a lot of common patterns for the Language unification. It's a living document we want to pull forward. The most recent version can be found in 2021-11-01-preview.

Reply:
(Sorry, just meant to comment - not approve.)

Reply:
Yep, I've just copied the one from a previous version and made a few additive changes to it in 2022-02-01-preview.

I am currently not able to use some of the definitions in the common.json, however.  For example, we use parameters `skip` and `top` for pagination, but in our service definition we specify default, min and max values for each.  This currently isn't being done in the common.json, and I don't want to add our settings to the common one in case this might interfere with other teams.  If its not required to set default, min and max values then I can use the common definition with the understanding that different services will set their own values.  Otherwise we need to have a bigger discussion across multiple teams about what those values should be and unify on that.

Reply:
Just synced with @johanste , and I now feel more comfortable using the common definitions of skip and top.  So now we are fully integrated (in as much as we can be) with common.json.  :)